### PR TITLE
Stardew Valley: Fix generation fail with SVE and entrance rando when Wizard Tower is in place of Sprite Spring

### PR DESCRIPTION
**Pull request by Jouramie**
## What is this fixing or adding?
`test_given_mod_names_when_generate_paired_with_entrance_randomizer_then_basic_checks` fails because
<details><summary>Assert all locations are accessible</summary>
<p>

```
AssertionError: False is not true : (Reach Location Complete Crafts Room & Reach Location Complete Pantry & Reach Location Complete Fish Tank & Reach Location Complete Bulletin Board & Reach Location Complete Vault & Reach Location Complete Boiler Room) -> False
  Reach Location Complete Crafts Room -> False
    (Received 9 Foraging Level & Has Fiddlehead Fern & Has Fiddlehead Fern & Reach Region Wizard Tower & Reach Region Wizard Tower & Received 3 [Has Dandelion -> Received Spring, Has Purple Mushroom -> (Reach Region The Mines - Floor 95 | Reach Region Skull Cavern Floor 25 | (Received Mushroom Boxes & Reach Region Farmcave) | (Received Fall & Reach Region Forest West) | Reach Region Sprite Spring Cave), Has White Algae -> (Received Progressive Fishing Rod & Reach Region The Mines - Floor 20), Has Blackberry -> (Received Fall | (Received Fruit Bats & Reach Region Farmcave))] & Has Fiddlehead Fern & Has Grape & Has Rainbow Shell & Reach Region Wizard Tower & Has Sea Urchin & Has Clam & Has Mussel & Has Oyster & Reach Region Wizard Tower & Has Clay & Has Hardwood & Has Wood & Has Fiber & Reach Region Wizard Tower & Reach Region Wizard Tower & Has Sap) -> False
      Reach Region Wizard Tower -> False
        Reach Entrance Forest West to Sprite Spring -> False
          Has Magic Ink -> False
            Reach Region Wizard Tower -> False
              Reach Entrance Forest West to Sprite Spring -> False
                Has Magic Ink -> False
                  Reach Region Wizard Tower -> False
                    Reach Entrance Forest West to Sprite Spring -> False
```

And it goes on, and on, and on...

</p>
</details> 

## How was this tested?
Hard code seed in the unit test. It passes with the fix.

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

File: worlds/stardew_valley/data/villagers_data.py [hunk 1]
```diff
@@ -366,10 +366,11 @@ def villager(name: str, bachelor: bool, locations: Tuple[str, ...], birthday: st
     return npc
 
 
-def make_bachelor(mod_name: str, npc: Villager):
+def adapt_wizard_to_sve(mod_name: str, npc: Villager):
     if npc.mod_name:
         mod_name = npc.mod_name
-    return Villager(npc.name, True, npc.locations, npc.birthday, npc.gifts, npc.available, mod_name)
+    # The wizard leaves his tower on sunday, for like 1 hour... Good enough to meet him!
```
Comment (agilbert1412) [inline worlds/stardew_valley/data/villagers_data.py:372]: lmao

**Review by agilbert1412 — APPROVED** [review]

File: worlds/stardew_valley/data/villagers_data.py (hunk 1 above)
Comment (Jouramie) [reply worlds/stardew_valley/data/villagers_data.py:372]: Thanks for the valuable feedback.

**Comment by ScipioWright** [comment]
I hope the test failure here isn't the one this PR was trying to fix?

**Comment by Jouramie** [comment]
It is not. This test seems to be failing randomly. That's an unrelated issue.

**Comment by BootsinSoots** [comment]
Fails on Windows 3.8 (But not Ubuntu?)

**Comment by Jouramie** [comment]
@BootsinSoots Unrelated. A random need failed for other reasons. The tests are all green 👍

**Comment by BootsinSoots** [comment]
Thanks, I was just double checking

**Review by black-sliver — APPROVED** [review]
code looks good

**Pull request merged**